csrankings.csv

```diff
@@ -735,6 +735,7 @@ Amelia Regan,University of California - Irvine,http://www.faculty.uci.edu/profil
 Amelie Chi Zhou,Shenzhen University,http://nhpcc.szu.edu.cn/zhouchi,pR-IEhcAAAAJ
 Amer Dawoud,University of Southern Mississippi,https://www.usm.edu/computing/faculty/amer-dawoud,2O9BsM0AAAAJ
 Amer E. Mouawad,American University of Beirut,http://www.aub.edu.lb/pages/profile.aspx?memberId=aa368,MK6WseUAAAAJ
+Amey Bhangale|University of California - Riverside|https://sites.google.com/view/amey-bhangale/home|ygJHy5UAAAAJ

```

Comment:
This and the others are using the wrong separator ("|" instead of ","). Please fix and I will merge.

Reply:
Oops, my bad. I was using a vim plugin to view CSV files as tables. I've fixed the issue now.